### PR TITLE
Update image.ts

### PR DESCRIPTION
**Pull request by aWeirdo**
Added image._markAsDirty(); on update to cellId
http://www.html5gamedevs.com/topic/34365-gui-imagecellid-doesnt-update-the-image/

### CODE REVIEW

File: gui/src/controls/image.ts [hunk 1]
```diff
@@ -174,6 +174,7 @@ module BABYLON.GUI {
         }
         set cellId(value: number) {
             this._cellId = value;
+            this._markAsDirty();
```
Comment (sebavan) [inline gui/src/controls/image.ts]: Could you check if the _cellId is different before (same pattern used elsewhere in the file like https://github.com/BabylonJS/Babylon.js/pull/3291/files#diff-6d9c4dadf5a981b552ad14470f429d4dR101)

Would be cool to set the same  pattern in cellWidth and cellHeight as well for consistency.

**Comment by aWeirdo** [comment]
Travis failed on https://github.com/BabylonJS/Babylon.js/pull/3291/commits/cdda8fc1497d6fbc06dc6cb46e2672c49af0632f .. not sure why

File: gui/src/controls/image.ts [hunk 2]
```diff
@@ -159,21 +159,34 @@ module BABYLON.GUI {
...
 
         get cellHeight(): number {
             return this._cellHeight;
         }
         set cellHeight(value: number) {
+            if(this._cellHeight === value){
+                return;
+            }
+            
             this._cellHeight = value;
```
Comment (sebavan) [inline gui/src/controls/image.ts]: missing the _markAsDirty here

File: gui/src/controls/image.ts [hunk 3]
```diff
@@ -159,21 +159,34 @@ module BABYLON.GUI {
             return this._cellWidth;
         }
         set cellWidth(value: number) {
+            if(this._cellWidth === value){
+                return;
+            }
+            
             this._cellWidth = value;
```
Comment (sebavan) [inline gui/src/controls/image.ts]: missing the _markAsDirty here

Comment (aWeirdo) [reply gui/src/controls/image.ts]: Didn't add it as it's usually not something changed on-the-go, but i suppose it's possible someone has different sized images in the same texture.

Comment (aWeirdo) [reply gui/src/controls/image.ts]: added.

**Pull request merged**